source/dotnet/subsystem-tests/Features/SettlementReports/SettlementReportJobConcurrentRunsScenario.cs

```diff
@@ -0,0 +1,112 @@
+﻿// Copyright 2020 Energinet DataHub A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License2");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Energinet.DataHub.Wholesale.SubsystemTests.Features.SettlementReports.Fixtures;
+using Energinet.DataHub.Wholesale.SubsystemTests.Features.SettlementReports.States;
+using Energinet.DataHub.Wholesale.SubsystemTests.Fixtures.Attributes;
+using Energinet.DataHub.Wholesale.SubsystemTests.Fixtures.LazyFixture;
+using FluentAssertions;
+using Xunit;
+
+namespace Energinet.DataHub.Wholesale.SubsystemTests.Features.SettlementReports;
+
+[Collection(nameof(SettlementReportJobCollectionDefinition))]
+[TestCaseOrderer(
+    ordererTypeName: "Energinet.DataHub.Wholesale.SubsystemTests.Fixtures.Orderers.ScenarioStepOrderer",
+    ordererAssemblyName: "Energinet.DataHub.Wholesale.SubsystemTests")]
+public class SettlementReportJobConcurrentRunsScenario : SubsystemTestsBase<SettlementReportJobScenarioFixture<ConcurrentRunsScenarioState>>
+{
+    public SettlementReportJobConcurrentRunsScenario(LazyFixtureFactory<SettlementReportJobScenarioFixture<ConcurrentRunsScenarioState>> lazyFixtureFactory)
+        : base(lazyFixtureFactory)
+    {
+    }
+
+    [ScenarioStep(0)]
+    [SubsystemFact]
+    public void Given_ScenarioSetup()
+    {
+        // Input
+        Fixture.ScenarioState.JobParametersTemplate = new[]
+        {
+          "--report-id=Guid",
+          "--calculation-type=wholesale_fixing",
+          $"--calculation-id-by-grid-area={{\"804\": \"{Fixture.Configuration.InputCalculationId}\"}}",
+          "--period-start=2023-01-31T23:00:00Z",
+          "--period-end=2023-02-28T23:00:00Z",
+          "--market-role=datahub_administrator",
+        };
+
+        // Expectations
+        Fixture.ScenarioState.ExpectedMaxConcurrentRuns = 40;
```

Comment:
It equally well be "expected minimum".
```suggestion
        Fixture.ScenarioState.ExpectedConcurrentRuns = 40;
```

Reply:
Fixed